### PR TITLE
merge duplicate tryton entries

### PR DESCRIPTION
**Pull request by nodiscc**
Tryton had 1 entry in ERP and 1 in Money/Budgeting/Management.

### CODE REVIEW

**Comment by danger-awesome-Selfhosted** [comment]
<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true" data-kind="Error">
          1 Error
      </th>
     </tr>
  </thead>
  <tbody>
    <tr>
      <td>:no_entry_sign:</td>
      <td data-sticky="false">Please provide a summary in the Pull Request description</td>
    </tr>
  </tbody>
</table>

<p align="right" data-meta="generated_by_danger">
  Generated by :no_entry_sign: <a href="http://danger.systems/">danger</a>
</p>

**Comment by danger-awesome-Selfhosted** [comment]
<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true" data-kind="Error">
          1 Error
      </th>
     </tr>
  </thead>
  <tbody>
    <tr>
      <td>:no_entry_sign:</td>
      <td data-sticky="false">Please provide a summary in the Pull Request description</td>
    </tr>
  </tbody>
</table>

<p align="right" data-meta="generated_by_danger">
  Generated by :no_entry_sign: <a href="http://danger.systems/">danger</a>
</p>

File: README.md [hunk 1]
```diff
@@ -495,7 +495,7 @@ See https://github.com/n1trux/awesome-sysadmin#dns
 
 * [ERPNext](https://erpnext.com) - Free open source ERP system. ([Demo](https://demo.erpnext.com), [Source Code](https://github.com/frappe/erpnext)) `GPLv3` `Python`
 * [Odoo](http://odoo.com) - Free open source ERP system. ([Demo](https://demo.odoo.com/), [Source Code](https://github.com/odoo/odoo)) `LGPLv3` `Python`
-* [Tryton](http://www.tryton.org/) - Free open source business solution. ([Demo](http://demo.tryton.org/), [Source Code](https://hg.tryton.org/)) `GPLv3` `Python`
+* [Tryton](http://www.tryton.org/) - Free open source business solution. ([Demo](http://demo.tryton.org/) admin/admin, [Source Code](https://hg.tryton.org/)) `GPLv3` `Python`
```
Comment (arylatt) [inline README.md]: Don't think we should really have demo creds inline... @Kickball?

Comment (arylatt) [inline README.md]: Can you replace the `Demo` link with `http://www.tryton.org/download.html`?

**Comment by nodiscc** [comment]
@arylatt Done, thank you, this should be the recommended method when linking to a demo that requires credentials (linking to the credentials page).

**Pull request merged**